### ISSUE TITLE
Request: buildah commit returns resulting image's ID on success

### ISSUE THREAD

**Issue by AndydeCleyre** [projectatomic/buildah#565]
**Description**

The idea is to make `buildah commit`'s behavior analogous to `buildah from`'s, regarding the return value, which is documented as:

```
The container ID of the container that was created. On error, -1 is returned and errno is returned.
```

So the proposed return value for `buildah commit` could be documented as:

```
The image ID of the image that was created. On error, -1 is returned and errno is returned.
```

**Output of `buildah version`:**

```
Version:       0.10
Go Version:    go1.9.2
Image Spec:    1.0.0
Runtime Spec:  1.0.0
Git Commit:    adfb256
Built:         Sat Jan 13 02:11:50 2018
OS/Arch:       linux/amd64
```

**Comment by TomSweeneyRedHat** [comment]
@AndydeCleyre thanks for the issue and the discussions in IRC.  Will take a look at this.  We're returning the imageId for the 'podman commit' command so should probably be doing the same here in Buildah.

**Comment by TomSweeneyRedHat** [comment]
Closed via #566
